dbms/src/Storages/Page/V3/tests/gtest_wal_log.cpp

```diff
@@ -303,6 +315,15 @@ TEST_P(LogFileRWTest, Fragmentation)
     ASSERT_EQ("EOF", read());
 }
 
+// This test may take a lot of time
+TEST_P(LogFileRWTest, DifferentPayloadSize)
+{
+    for (size_t i = 0; i < 40000; i += 1)
+    {
+        write(repeatedString("a", i));
+    }
+}
```

Comment:
This slow down the ut running time in CI from about 7 min to 17 min...

Reply:
Can we ease this test after we keep the buffer size unchanged?

Reply:
Disabled test test.